npep/npep-133.md

```diff
@@ -0,0 +1,140 @@
+# NPEP-133: FQDN Selector for Egress Traffic
+
+* Issue:
+  [#133](https://github.com/kubernetes-sigs/network-policy-api/issues/133)
+* Status: Provisional
+
+## TLDR
+
+This enhancement proposes adding a new optional selector to specify egress peers
+using [Fully Qualified Domain Names](https://www.wikipedia.org/wiki/Fully_qualified_domain_name)
+(FQDNs).
+
+## Goals
+
+* Provide a selector to specify egress peers using a Fully Qualified Domain Name
+  (for example `kubernetes.io`).
+* Support a restricted set of regex matching capabilities when specifying FQDNs.
+* Currently only `ALLOW` type rules are proposed.
+  * Correctly enforcing DENY rules based on FQDN selectors is difficult as there
+    is no guarantee a Network Policy plugin is aware of all IPs backing a FQDN
+    policy. 
+* Currently only AdminNetworkPolicy is the intended scope for this proposal.
+  * Since Kubernetes NetworkPolicy does not have a FQDN selector, adding this
+    capability to BaselineAdminNetworkPolicy can result in unintended behavior.
```

Comment:
```suggestion
  * Since Kubernetes NetworkPolicy does not have a FQDN selector, adding this
    capability to BaselineAdminNetworkPolicy would not be very useful.
```

Saying that it "can result in unintended behavior" makes it sound like there's some tricky unexpected interaction, but that's not the problem. The problem is just that all of the good user stories that would make use of FQDN BANPs only work if you have FQDN NPs too.

Reply:
Yeah, good point. Reworded